encrypted-media-respec.html

```diff
@@ -2355,8 +2390,13 @@
           <ol>
             <li><p>Let the <var>media element</var> be the specified <a def-id="htmlmediaelement"></a> object.</p></li>
             <li><p>If the <var>media element</var>'s <var>waiting for key</var> is <code>false</code>, abort these steps.</p></li>
-            <li><p>Attempt to resume the <a def-id="resource-fetch-algorithm"></a> by running the <a def-id="encrypted-block-encountered-algorithm"></a> algorithm.</p></li>
-            <li><p>If the user agent can advance the <a def-id="current-playback-position"></a> in the <a def-id="direction-of-playback"></a>, set the <var>media element</var>'s <var>waiting for key</var> value to <code>false</code>.</p></li>
+            <li><p>Run the <a def-id="attempt-to-decrypt-algorithm"></a> algorithm on the <var>media element</var>.</p></li>
```

Comment:
The intro paragraph to this algorithm is not accurate. It can also be called from setMediaKeys() to unblock the wait in the Initialization Data Encountered algorithm.